server/views/challenges/showHTML.jade

```diff
@@ -24,15 +24,14 @@ block content
                                         p.wrappable!= sentence
                                 .negative-bottom-margin-30
                     .button-spacer
-                    .btn-big.btn.btn-primary.btn-block#submitButton
+                    button.btn-big.btn.btn-primary.btn-block#submitButton
                         | Run tests (ctrl + enter)
                     .button-spacer
-                    .btn-group.input-group.btn-group-justified
-                        label.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-reset-modal Reset
-                        label.btn.btn-primary.btn-primary-ghost.hidden-sm.hidden-md.hidden-lg
-                            a(href='//gitter.im/freecodecamp/help') Help
-                        label.btn.btn-primary.btn-primary-ghost.hidden-xs.btn-lg#challenge-help-btn Help
-                        label.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-issue-modal Bug
+                    .btn-group.btn-group-justified
+                        a.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-reset-modal(href="#" role="button") Reset
+                        a.btn.btn-primary.btn-primary-ghost.hidden-sm.hidden-md.hidden-lg(href='//gitter.im/freecodecamp/help') Help
+                        a.btn.btn-primary.btn-primary-ghost.hidden-xs.btn-lg#challenge-help-btn(href="#" role="button") Help
+                        a.btn.btn-primary.btn-primary-ghost.btn-lg#trigger-issue-modal(href="#" role="button") Bug
```

Comment:
I guess this `jade` template can be in its own partial, but they are not exactly same.
However I could not find any impact by making them same. ***Please note I have tested on in one screen resolution***.

But if someone confirms, may I can update the PR.